examples/python/arkit_scenes/arkit_scenes/download_dataset.py

```diff
@@ -196,7 +196,7 @@ def download_laser_scanner_point_clouds(laser_scanner_point_cloud_id: str, visit
         download_file(file_url, filename, laser_scanner_point_clouds_folder_path)
 
 
-def get_metadata(dataset: str, download_dir: Path) -> Optional[pd.DataFrame]:
```

Comment:
I think this will break `Python 3.9`, so we should do this first: 
* https://github.com/rerun-io/rerun/issues/10958

Reply:
Oh, you're right. For a moment I thought python 3.9 did support those (I mixed up with `ThingLike = Thing | Stuff` type variable). ruff should know better though. There must be something wrong in the configuration file.

That said, the earlier we get rid of 3.9 the better in my book, because it gets us ✨`match` ✨